crates/bevy_ecs/src/query/state.rs

```diff
@@ -289,6 +340,10 @@ where
         QueryCombinationIter::new(world, self, last_change_tick, change_tick)
     }
 
+    /// Runs `f` on each query result for the given [`World`]. This is faster than the equivalent
```

Comment:
```suggestion
    /// Runs `func` on each query result for the given [`World`]. This is faster than the equivalent
```

Reply:
This also applies to each other place where `func` is used as the parameter name. 
`for_each_mut`
`par_for_each`
`par_for_each_mut`
etc...